packages/cozy-client/src/CozyClient.js

```diff
@@ -693,7 +693,12 @@ client.query(Q('io.cozy.bills'))`)
       doctypeVersion = get(schema, 'doctypeVersion')
     }
 
-    const { slug, sourceAccount, version } = this.appMetadata
+    const {
+      slug,
+      sourceAccount,
+      sourceAccountIdentifier,
+      version
+    } = this.appMetadata
```

Comment:
Don't we need to update the appMetadata type? If it's an any, maybe we should type it.

Reply:
Is it documented somewhere to pass the sourceAccount and sourceAccountIdentifier? If no, you know my point ;)

Reply:
Enhanced  appMetadata type documentation + updated cozy-client metadata documentation with sourceAccountIdentifier